### PR TITLE
Documentation modification and addition for linkage disequilibrium.

### PR DESCRIPTION
**Pull request by cmclean**
Mostly additions and modifications to include the LD project, with a few typo fixes.

### CODE REVIEW

**Comment by deflaux** [comment]
Great docs @cmclean !

http://googlegenomics.readthedocs.org/en/staging/use_cases/linkage_disequilibrium/index.html

File: docs/source/use_cases/discover_google_data/1000_genomes_ld.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_google_data/1000_genomes_ld.html
```
Comment (pgrosu) [inline docs/source/use_cases/discover_google_data/1000_genomes_ld.rst]: The updates look nice, but when I click on the following link it says *"SORRY This page does  not exist yet"* - is there a certain period of time I should wait?

http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_google_data/1000_genomes_ld.html

Thanks,
~p

File: docs/source/use_cases/discover_google_data/genomic_data_toc.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_google_data/genomic_data_toc.html
```
Comment (pgrosu) [inline docs/source/use_cases/discover_google_data/genomic_data_toc.rst]: Same thing with this link :(

File: docs/source/use_cases/discover_google_data/1000_genomes_ld.rst (hunk 1 above)
Comment (mbookman) [inline docs/source/use_cases/discover_google_data/1000_genomes_ld.rst]: This pull request is into the "staging" branch. The github link explicitly goes to "latest".
Once this PR is merged into latest, then the link will work.

Comment (pgrosu) [inline docs/source/use_cases/discover_google_data/1000_genomes_ld.rst]: Ah, ok - thanks Matt.  I'll wait until merged into latest.

Thanks,
~p

**Pull request merged**